src/Boxer/Platform/Win32/Win32Application.cpp

```diff
@@ -110,14 +133,39 @@ namespace boxer {
 			const char* errorStr = (const char*)glewGetErrorString(err);
 			ASSERT(0, errorStr);
 		}
+
+		// This looks nasty
+		// TODO: Migrate this
+		WGLSWAPINTERVALEXT* wglSwapIntervalEXT = (WGLSWAPINTERVALEXT*)wglGetProcAddress("wglSwapIntervalEXT");
+		wglSwapIntervalEXT(0);
 	}
 
 	void Application::Launch() {
 		Win32Handle* win32 = reinterpret_cast<Win32Handle*>(_Handle);
 
-		// Event handling
+		// Initialize Timer
+		LARGE_INTEGER frequency;
+		QueryPerformanceFrequency(&frequency);
+		m_Frequency = 1.0 / F64(frequency.QuadPart);
+		QueryPerformanceCounter(&m_Start);

```

Comment:
Not usually a fan of this being done inline. Perhaps clean this up by moving it to a private method and calling it instead. Launch should be as simple and lightweight as possible.